isb_lib/models/isb_core_record.py

```diff
@@ -0,0 +1,255 @@
+from typing import Optional
+
+from sqlmodel import SQLModel, Field
+
+
```

Comment:
This ends up being pretty much as you'd expect -- one field == one column in the SQLite schema.